tough/src/cache.rs

```diff
@@ -42,7 +42,7 @@ impl<'a, T: Transport> Repository<'a, T> {
                 self.cache_target(&targets_outdir, target_name.as_ref())?;
             }
         } else {
-            let targets = &self.targets.signed.targets_map();
+            let targets = (*self.targets.signed).targets_map();
```

Comment:
Okay yeah this is weird, I need to play with this. I don't think I expected Deref coercion to not work...

Reply:
In your mind it would be transparent to all the call sites? Is that the issue?

Reply:
Deref coercion only works on methods, not fields.